src/main/datatypes/MQ2WindowType.cpp

```diff
@@ -630,6 +669,15 @@ bool MQ2WindowType::GetMember(MQVarPtr VarPtr, const char* Member, char* Index,
 		return false;
 	}
 
+	case WindowMembers::InvSlot:
+		if (pWnd->GetType() == UI_InvSlot)
+		{
+			Dest.Type = pInvSlotWindowType;
+			Dest.Ptr = VarPtr.Ptr;
```

Comment:
I'd prefer use of Dest.Set here, this is the old method that we don't need to be using anymore for most things

Reply:
On second thought, this is actually fine -- we don't own the CXWnd pointer, so you'd also need to create a wrapper around it.... not work that needs to be done in this PR